bootstrap.sh

```
@@ -242,6 +241,195 @@ function getUbuntuVersion {
+        if [ -x "${PYTHON3_BIN}" ]; then
+            log "Symlinking python3 (${PYTHON3_BIN}) to (${PYTHON_BIN})..."
+            ln -s "${PYTHON3_BIN}" "${PYTHON_BIN}"
+        fi
+    fi
+}
+
+function stepInstallZshPrezto() {
+    # https://github.com/sorin-ionescu/prezto
+    logStep "Installing ZSH and Prezto..."
+    runCmdAndLog ${APT_INSTALL} zsh
+    local ZSH_BIN=$(command -v zsh)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)